### PR TITLE
Describe the Startup Page block options from Advanced > Admin page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- describes the Startup Page block options from Advanced > Admin page
- edits the Admin User Emails block

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/advanced/admin.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=My%20Account

whatsnew
Added the new _Startup Page options_ section in the [Admin configuration](https://docs.magento.com/user-guide/configuration/advanced/admin.html) page to list and describe the options for choosing a startup page in the Admin.

### CODE REVIEW

File: src/configuration/advanced/admin.md [hunk 1]
```diff
@@ -24,6 +24,48 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Adva
...
+|[Content]({% link content.md %}) | [Elements]({% link cms/content-elements.md %}) | [Pages]({% link cms/pages.md %})<br/><span class="ee-only">[Hierarchy]({% link cms/page-hierarchy.md %})</span><br/>[Blocks]({% link cms/blocks.md %})<br/><span class="ee-only">[Dynamic Blocks]({% link cms/dynamic-blocks.md %})</span><br/>[Widgets]({% link cms/widgets.md %})<br/>[Media Gallery]({% link cms/media-storage.md %})||
+||[Design]({% link design/design-theme.md %}) | [Configuration]({% link design/configuration.md %})<br/>[Themes]({% link design/themes.md %})<br/>[Schedule]({% link design/schedule.md %})||
+||<span class="ee-only">[Content Staging]({% link cms/content-staging.md %})</span><br />
+|[Reports]({% link reports.md %}) | [Marketing]({% link reports/marketing-reports.md %})|Products in Cart<br />Search Terms<br />Abandoned Carts<br />Newsletter Problem Reports||
+||[Reviews]({% link reports/review-reports.md %})|By Customer<br />By Products<br />Yotro Reviews|
```
Comment (leslietilling) [inline src/configuration/advanced/admin.md:46]:
```suggestion
||[Reviews]({% link reports/review-reports.md %})|By Customer<br />By Products<br />Yotpo Reviews|
```

File: src/configuration/advanced/admin.md [hunk 2]
```diff
@@ -24,6 +24,48 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Adva
...
+||[Sales]({% link reports/sales-reports.md %})||
+||<span class="ee-only">System Insights|[Site-Wide Analysis Tool]({% link reports/site-wide-analysis-tool.md %})|
+||[Customers]({% link reports/customer-reports.md %})| Order Total<br/>Order Count<br/>New<br/>Wish Lists<br/>Segments<br/>|
+||[Products]({% link reports/product-reports.md %})|Views<br/>Bestsellers<br/>Low Stock<br/>Ordered<br/>Downloads|
+||<span class="ee-only">[Private Sales]({% link marketing/events-private-sales.md %})|Invitations<br/>Invited Customers<br/>Conversions|
+||[Statistics]({% link reports/statistics.md %})|Refresh Statistics|
+||[Business Intelligence]({% link reports/business-intelligence.md %})|Advanced Reporting<br/>BI Essentials<br/>Yotro Analytics|
```
Comment (leslietilling) [inline src/configuration/advanced/admin.md:53]:
```suggestion
||[Business Intelligence]({% link reports/business-intelligence.md %})|Advanced Reporting<br/>BI Essentials<br/>Yotpo Analytics|
```

File: src/configuration/advanced/admin.md [hunk 3]
```diff
@@ -24,6 +24,48 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Adva
...
+||[Inventory]({% link configuration/catalog/inventory.md %})|[Sources]({% link catalog/sources.md %})<br/>Stocks||
+||[Taxes]({% link tax/taxes.md %})|[Tax Rules]({% link tax/tax-rules.md %})<br/>[Tax Zones and Rates]({% link tax/tax-zones-rates.md %})|
+||[Currency]({% link stores/currency.md %})|[Currency Rates]({% link stores/currency-configuration.md %})<br/>[Currency Symbols]({% link stores/currency-symbols.md %})||
+||[Attributes]({% link stores/attributes.md %})|[Product]({% link stores/attributes-product.md %})<br/>[Update Attributes]({% link stores/attribute-product-create.md %})<br/>[Attribute Set]({% link stores/attribute-sets.md %})<br/>[Ratings]({% link stores/attributes-rating.md %})|
+||Other Settings|[Reward ExchangeRates]({% link marketing/reward-exchange-rates.md %})<br/>[Gift Wrapping]({% link sales/gift-wrap.md %})<br/>[Gift Registry]({% link customers/account-dashboard-gift-registry.md %})|
```
Comment (leslietilling) [inline src/configuration/advanced/admin.md:59]:
```suggestion
||Other Settings|[Reward Exchange Rates]({% link marketing/reward-exchange-rates.md %})<br/>[Gift Wrapping]({% link sales/gift-wrap.md %})<br/>[Gift Registry]({% link customers/account-dashboard-gift-registry.md %})|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
A few minor edits

**Comment by ZhovnirMaria** [comment]
@leslietilling 
Thank you! Done ✅

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1224`.

**Pull request merged**